org.eclipse.wb.core/src/org/eclipse/wb/internal/core/utils/dialogfields/LayoutDialogFieldGroup.java

```diff
@@ -284,7 +284,7 @@ public void enableSelectionButton(int index, boolean enable) {
     }
   }
 
-  /*(non-Javadoc)
+  /*
    * @see org.eclipse.jdt.internal.ui.wizards.dialogfields.DialogField#refresh()

```

Comment:
The whole block can go, as we have @Override. Please move this also to a new PR so that this PR only contains the function changes.